src/main/java/utils/UserFactory.java

```diff
@@ -0,0 +1,20 @@
+package utils;
+
+import com.github.javafaker.Faker;
+
+public class UserFactory {
```

Comment:
The whole idea of this class is to store/manage credentials and users. You don't need any static blocks for that. You can just create getExistingUser() method and return existing user object there.